osquery/tables/system/windows/userassist.cpp

```diff
@@ -0,0 +1,143 @@
+/**
+ *  Copyright (c) 2014-present, Facebook, Inc.
+ *  All rights reserved.
+ *
+ *  This source code is licensed in accordance with the terms specified in
+ *  the LICENSE file found in the root directory of this source tree.
+ */
+
+#include <osquery/core.h>
+#include <osquery/tables.h>
+#include <osquery/tables/system/windows/registry.h>
+#include <osquery/utils/conversions/split.h>
+#include <osquery/utils/system/time.h>
+#include <string>
+
+namespace osquery {
+namespace tables {
+
+// Decode ROT13 sub key value
+std::string rot_decode(std::string value_key_reg) {
+  std::string decoded_value_key;
+
+  for (size_t i = 0; i < value_key_reg.size(); i++) {
+    if (isalpha(value_key_reg[i])) {
+      if (value_key_reg[i] >= 'a' && value_key_reg[i] <= 'm') {
+        decoded_value_key.append(1, value_key_reg[i] + 13);
+      } else if (value_key_reg[i] >= 'm' && value_key_reg[i] <= 'z') {
+        decoded_value_key.append(1, value_key_reg[i] - 13);
+      } else if (value_key_reg[i] >= 'A' && value_key_reg[i] <= 'M') {
+        decoded_value_key.append(1, value_key_reg[i] + 13);
+      } else if (value_key_reg[i] >= 'M' && value_key_reg[i] <= 'Z') {
+        decoded_value_key.append(1, value_key_reg[i] - 13);
+      }
+    } else {
+      decoded_value_key.append(1, value_key_reg[i]);
+    }
+  }
+  return decoded_value_key;
+}
+
+QueryData genUserAssist(QueryContext& context) {
+  QueryData results;
+  QueryData users;
+
+  queryKey("HKEY_USERS", users);
+
+  for (const auto& uKey : users) {
+    if (uKey.count("type") == 0 || uKey.count("path") == 0) {
+      continue;
+    }
+
+    std::string path = uKey.at("path");
+    std::string fullPath = path.append(
+        "\\SOFTWARE\\Microsoft\\Windows\\CurrentVersion\\Explorer\\UserAssist");
```

Comment:
If this is a static value I recommend breaking it out into a `constexpr std::string kUserAssistLocation`.